code/game/objects/effects/decals/remains.dm

```diff
@@ -22,12 +22,6 @@
 	icon_state = "remainsrobot"
 	anchored = TRUE
 
-/obj/effect/decal/remains/robot/New()
-	..()
-	var/datum/effect_system/spark_spread/s = new /datum/effect_system/spark_spread

```

Comment:
Why it did this is beyond me. Of all the time that robot remains are created, sparks are never really justified.

Reply:
I mean, robots dying and spreading sparks on death makes sense to me.